charts/stable/frigate/values.yaml

```
@@ -0,0 +1,106 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: blakeblackshear/frigate
+  # -- image tag
+  tag: 0.8.4-amd64
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- environment variables. See [docs](https://blakeblackshear.github.io/frigate/) for more details.
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+  # -- Set a RTSP password
+  # @default -- empty
+  FRIGATE_RTSP_PASSWORD: ""
+
+# -- Configures service settings for the chart.
+# @default -- See values.yaml
+service:
+  main:
+    ports:
+      http:
+        port: 5000
+      rtmp:
+        enabled: false
+        port: 1935
+        protocol: TCP
+        targetPort: 1935
+
+ingress:
+  # -- Enable and configure ingress settings for the chart under this key.
+  # @default -- See values.yaml
+  main:
+    enabled: false
+
+securityContext:
+  # -- (bool) Privileged securityContext may be required if USB devices are accessed directly through the host machine
+  privileged: true
```

```suggestion
  privileged:  # true
```
Missed this one earlier, in other charts we have the user explicitly set this:
https://github.com/k8s-at-home/charts/blob/09ab3726253bffb2b9bd7f6afc8bb129d0a4d285/charts/stable/zigbee2mqtt/values.yaml#L38

---

I just set it to false by default, that ok?